stripe/test/helper.py

```diff
@@ -12,16 +12,12 @@
 
 NOW = datetime.datetime.now()
 
-DUMMY_CARD = {
-    'number': '4242424242424242',
-    'exp_month': NOW.month,
-    'exp_year': NOW.year + 4
-}
-DUMMY_DEBIT_CARD = {
-    'number': '4000056655665556',
-    'exp_month': NOW.month,
-    'exp_year': NOW.year + 4
-}
+DUMMY_CARD = 'tok_visa'
+
+DUMMY_DEBIT_CARD = 'tok_visa_debit'
+
+DUMMY_EXPIRED_CARD = 'tok_chargeDeclinedExpiredCard'
```

Comment:
Could we just suffix these with `_TOKEN` so that when you're reading the referencing code it's obvious that it's a token and not some kind of card hash?